<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>salome2ccx</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>9</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QPushButton" name="pb_sel_mesh">
     <property name="text">
      <string>Select meshes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="le_selectMesh">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_output_folder">
     <property name="text">
      <string>Select output folder</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="le_output_folder">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Choose converter:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="converter">
     <item>
      <property name="text">
       <string>unv2ccx</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>unical3</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="rb_delet_e_f">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Delete edges and faces</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="rb_cgx">
     <property name="text">
      <string>Open each result in CGX</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="okbox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
